haskoin-wallet/Network/Haskoin/Wallet/Client/Commands.hs

```diff
@@ -832,8 +800,8 @@ printAccount JsonAccount{..} = unlines $
     , "Gap     : " ++ show jsonAccountGap
     ]
     ++
-    [ "Deriv   : " ++ pathToStr d
-    | d <- maybeToList jsonAccountDerivation
+    [ "Index   : " ++ show (xPrvChild m)
+    | m <- maybeToList jsonAccountMaster -- TODO: How about for read-only ?
```

Comment:
If there is a single public key, it is also possible to obtain this number unambiguously.